### DISCUSSION TITLE
Why on my prompt does it say 'on ☁️'?

### DISCUSSION THREAD

**Discussion by HarrisonHemstreet** [starship/starship#3308] [Q&A] [answered]
Why on my prompt does it say 'on ☁️'?
![Screenshot from 2021-12-10 11-03-57](https://user-images.githubusercontent.com/65803282/145620763-1a1e38d0-df68-4715-927e-d3e3a17aed70.png)

**Comment by chipbuster** [comment, marked as answer]
You can ask your prompt with `starship explain`. This is *most likely* due to an AWS config being detected, but could be other things as well.

**Reply by HarrisonHemstreet** [reply]
![Screenshot from 2021-12-10 23-18-15](https://user-images.githubusercontent.com/65803282/145666550-e623ada8-9107-46de-8bad-c8da805f124b.png)

**Reply by HarrisonHemstreet** [reply]
Thanks for your response! :)
Unfortunately, I'm not seeing much information from `starship explain`. Any more ideas?

**Reply by chipbuster** [reply]
Well that's unusual.

If you haven't made any changes to your config, then this can be one of three things:
  - AWS
  - Google Cloud
  - OpenStack

You can try setting `export STARSHIP_LOG=debug` in order to get more logging info (if you're using zsh or bash, you'll need to tell me what shell you're using for syntax in other shells) and paste the log here---I'll see if there's anything illuminating.

**Reply by HarrisonHemstreet** [reply]
Looks like it just got worse after globally installing node!
![Screenshot from 2021-12-22 17-07-01](https://user-images.githubusercontent.com/65803282/147168668-ab3520e1-5547-4568-babe-f1dac01b08e1.png)

**Reply by HarrisonHemstreet** [reply]
I'm on bash btw

**Reply by HarrisonHemstreet** [reply]
Do you happen to know how I can uninstall and then reinstall Starship? maybe that would work?

**Reply by matchai** [reply]
As shown in your screenshot, the cloud emoji is for the [GCP module](https://starship.rs/config/#google-cloud-gcloud), which shows when you have an active GCloud config in `~/.config/gcloud/active_config` or when the `CLOUDSDK_CONFIG` environment variable is set.

As suggested by chipbuster, you may get more details on why it's appearing by setting `export STARSHIP_LOG=debug`.

Details on how to uninstall starship [can be found here](https://starship.rs/faq/#how-do-i-uninstall-starship), though Starship is stateless. Uninstalling and reinstalling shouldn't have any effect on what the prompt shows.

**Reply by HarrisonHemstreet** [reply]
Thank you! Your suggestion of looking into what I have stored in my .config directory worked! Thank you! :100:
